Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -221,9 +221,21 @@ protected virtual void Dispose(bool disposing)
         /// </summary>
         private void InitializeForFrameworkEventSource()
         {
-#if !NET40
-            FrameworkHttpProcessing frameworkHttpProcessing = new FrameworkHttpProcessing(this.telemetryConfiguration, DependencyTableStore.Instance.WebRequestCacheHolder, this.SetComponentCorrelationHttpHeaders, this.ExcludeComponentCorrelationHttpHeadersOnDomains, this.EffectiveProfileQueryEndpoint);
-            this.httpDesktopDiagnosticSourceListener = new HttpDesktopDiagnosticSourceListener(frameworkHttpProcessing);
+#if NET45
+            DesktopDiagnosticSourceHttpProcessing desktopHttpProcessing = new DesktopDiagnosticSourceHttpProcessing(
```

Comment:
we are now enabling both diagnostic source and framework event source. where is the logic to prevent double collection or only one will be active at a time?